external/mbedtls/debug.c

```diff
@@ -81,17 +81,13 @@ static inline void debug_send_line(const mbedtls_ssl_context *ssl, int level, co
 	 * Since there is no portable way to get one, use the address of the ssl
 	 * context instead, as it shouldn't be shared between threads.
 	 */
-#if 0							/* For TinyAra */
 #if defined(MBEDTLS_THREADING_C)
```

Comment:
Is it really required to make this API as inline function ? it's been called from several functions and it just increases code space.  May not be directly linked to your changes but am just mentioning the overhead of removing #if 0

Reply:
@pradeepns85 
if you see line 103 in debug.c, it always checks 'ssl->conf->f_dbg' to be registered by users.
As following this logic, even if an user registered f_dbg, it doesn't work and wastes code space. 
Moreover, if 'f_dbg' is not set, printing debugging logs fails.